publish-service/src/main/java/com/ericsson/eiffel/remrem/publish/config/SecurityConfig.java

```diff
@@ -52,25 +58,49 @@ public class SecurityConfig extends WebSecurityConfigurerAdapter {
 
     @Value("${activedirectory.userSearchFilter}")
     private String userSearchFilter;
-    
+
     @Value("${activedirectory.rootDn}")
     private String rootDn;
 
-    @Override
-    protected void configure(AuthenticationManagerBuilder auth) throws Exception {
+    @Value("${activedirectory.connectionTimeOut:#{0}}")
+    private Integer ldapTimeOut;
```

Comment:
I'd initialize ldapTimeOut to default value here, during the declaration. Then getTimeOut() can be simplified. The same for other properties having a default value.